tools/rna_tools/intarna/test-data/intarna_result.csv

```diff
@@ -0,0 +1,5 @@
+id1;start1;end1;id2;start2;end2;subseqDP;hybridDP;E
```

Comment:
@zwanli can we change the output to a tab separated file? Simply add a `sed` after the intarna command and change this file? CSV files are not really useful in the Galaxy context.